### ISSUE TITLE
Android Keyboard Shortcuts

### ISSUE THREAD

**Issue by locuturus** [maxim-saplin/CrossPlatformDiskTest#54]
Hello, I was testing this on my Pixel C the other day and the interface suggests hardware keyboard keys should be available as shortcuts but typing `d` for database or `o` for options didn't do anything with the Pixel C's Bluetooth keyboard.

**Comment by maxim-saplin** [comment]
Indeed, those letters are keyboard shortcuts and they work on Desktop platforms (Windows, macOS) from wher CPDT came to Android. Though keyboard support for Android was never implemented. Will keep the issue open, might come to implementing the feature some time in the future
